index.html

```diff
@@ -4,9 +4,167 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./styles.css">
     <title>Douglas' Portifolio page</title>
 </head>
 <body>
-    <h2>Portifolio</h2>
+<div class="main-layout">
+
+    <nav>
+        <span>Luzinda Douglas</span> 
+        <img src="https://img.icons8.com/ios-glyphs/30/ffffff/menu--v1.png" alt="Hamburger menu icon">  
+    </nav>  
```

Comment:
Kindly wrap this section in a `header` tag 😀

- _Typically the header tag is used as a page header, it contains the site's title or logo (or both) and the main site navigation. [Check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/header) for more info._